spring-batch-docs/asciidoc/readersAndWriters.adoc

```diff
@@ -989,10 +990,10 @@ public PatternMatchingCompositeLineMapper orderFileLineMapper() {
 }
 ----
 
-In this example, "LINEA" and "LINEB" have separate `LineTokenizers`, but they both use
+In this example, "LINEA" and "LINEB" have separate ``LineTokenizer``s, but they both use
 the same `FieldSetMapper`.
 
-The `PatternMatchingCompositeLineMapper` makes use of the `PatternMatcher's` match method
+The `PatternMatchingCompositeLineMapper` makes use of the ``PatternMatcher``'s match method
```

Comment:
For some reason, asciidoctor does not render correctly the following: `PatternMatcher`'s . I found that double backticks fix the rendering issue. You may see this in other places of the PRs.